src/index.js

```diff
@@ -31,11 +31,31 @@ class DynamicOverflow extends React.Component {
 
   calculateSize = () => {
     const containerWidth = this.containerNode && this.containerNode.clientWidth;
-    const firstChildWidth = this.tabNode && this.tabNode.clientWidth;
 
-    const maximumChildrenAllowed =
-      Math.floor(containerWidth / firstChildWidth) - 1;
-    const currentChildrenCount = this.props.list({}).length;
+    let moreNodeWidth = 0;
+    if (this.moreNode) {
+      const moreNodeStyle = window.getComputedStyle(this.moreNode);
+        const moreNodeMarginWidth = parseInt(moreNodeStyle.marginLeft) + parseInt(moreNodeStyle.marginRight);
+        moreNodeWidth = this.moreNode.offsetWidth + moreNodeMarginWidth;
+    }
```

Comment:
We can DRY up the logic to calculate a node's width:

```js
calculateNodeWidth = node => {
    const nodeStyle = window.getComputedStyle(node);
    const nodeMarginWidth = parseInt(nodeStyle.marginLeft) + parseInt(nodeStyle.marginRight);
    return node.offsetWidth + nodeMarginWidth;
}
```

Reply:
It looks like `node.offsetWidth` includes `padding` into the calculation, but not `margin`.

Are there any scenarios where we **must** use `margin`? I'm wondering if we can avoid this `getComputedStyle` call.